console-api/src/common.rs

```diff
@@ -61,6 +61,31 @@ impl<'a> From<&'a std::panic::Location<'a>> for Location {
     }
 }
 
+impl fmt::Display for field::Value {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            field::Value::BoolVal(v) => write!(f, "{}", v)?,
+            field::Value::StrVal(v) => write!(f, "{}", v)?,
+            field::Value::U64Val(v) => write!(f, "{}", v)?,
+            field::Value::DebugVal(v) => write!(f, "{}", v)?,
+            field::Value::I64Val(v) => write!(f, "{}", v)?,
```

Comment:
nit: it might be slightly simpler to just call the inner value's `fmt::Display` impl directly, rather than using the `write!` macro...i think there may be a very small amount of additional overhead introduced by going through a `format_args!`, and also i think this will ensure that any format parameters (e.g. padding/width/etc) are passed through to the inner value
```suggestion
            field::Value::BoolVal(v) => fmt::Display::fmt(v, f)?,
            field::Value::StrVal(v) => fmt::Display::fmt(v, f)?,
            field::Value::U64Val(v) => fmt::Display::fmt(v, f)?,
            field::Value::DebugVal(v) => fmt::Display::fmt(v, f)?,
            field::Value::I64Val(v) => fmt::Display::fmt(v, f)?,
```